neofetch

```diff
@@ -4557,6 +4568,7 @@ TEXT FORMATTING:
     --underline_char char       Character to use when underlining title
     --bold on/off               Enable/Disable bold text
     --separator string          Changes the default ':' separator to the specified string.
+    --align_output              Aligns the text output to the separator string
```

Comment:
Maybe add (left|right|off) here somewhere (if possible).

Reply:
Done in new force-push.